terraform/README.md

```diff
@@ -54,46 +53,61 @@ git clone https://github.com/NASA-PDS/nucleus.git
-    - mwaa_dag_s3_bucket_name = "pds-nucleus-airflow-dags-bucket-mcp-dev"
+    - env        : Name of the Cloud environment to deploy PDS Nucleus (E.g: "mcp-dev", "mcp-test")
+    - region     : AWS Region
+    - vpc_id     : VPC ID of your AWS VPC
+    - subnet_ids : List of Private Subnet IDs to be used for the MWAA
+    - vpc_cidr   : VPC CIDR for MWAA (E.g.: "10.1.0.0/16")
+    - pds_nucleus_opensearch_auth_config_file_path : Location of the OpenSearch auth file
+    - pds_nucleus_opensearch_url                   : OpenSearch URL specific for the PDS Node
+    - pds_node_name                                : PDS Node name
+    - pds_nucleus_harvest_replace_prefix_with      : Prex to replace in PDS Harvest tool
+    - mwaa_dag_s3_bucket_name         : S3 Bucket name to keep Airflow DAG files (E.g.: pds-nucleus-airflow-dags-bucket-mcp-test)
+    - pds_nucleus_staging_bucket_name : S3 Bucket name to keep PDS staging data files (E.g.: pds-nucleus-staging-mcp-test)
+    - pds_nucleus_config_bucket_name  : S3 Bucket name to keep temporary configurations (E.g.: pds-nucleus-config-mcp-test)
+    - pds_nucleus_default_airflow_dag_id : The default example DAG to be included for testing (E.g.: pds-basic-registry-load-use-case)
+
 
 > Note: `terraform.tfvars` is only used to test with your configuration with the actual values in your AWS account. This file will not be uploaded to GitHub as it's ignored by Git. Once testing is completed successfully work with your admin to get the values for these tested variables updated via GitHub secrets, which are dynamically passed in during runtime.
 
 ```
 # Example terraform.tfvars
 
-region = "us-west-2"
-vpc_id = "vpc-12345678"
-subnet_ids = ["subnet-12121212121","subnet-32323232323223"]
-vpc_cidr = "10.1.0.0/16"
-nucleus_security_group_ingress_cidr = ["0.0.0.0/0"]
-airflow_execution_role = "arn:aws:iam::<account-id>:role/pds-nucleus-mwaa-execution-role"
-efs_file_system_id = "fs-123232"
-registry_loader_scripts_access_point_id = "fsap-2121213232"
-registry_loader_default_configs_access_point_id = "fsap-1212323"
-task_role_arn = "arn:aws:iam::<account-id>:role/pds-nucleus-mwaa-execution-role"
-execution_role_arn = "arn:aws:iam::<account-id>:role/pds-nucleus-mwaa-execution-role"
-mwaa_dag_s3_bucket_name = "pds-nucleus-airflow-dags-bucket-mcp-dev"
+env        = "mcp-test"
+region     = "us-west-2"
+vpc_id     = "vpc-12345678"
+subnet_ids = ["subnet-123456789", "subnet-987654321"]
+vpc_cidr   = "10.2.0.0/16"
+
+pds_nucleus_opensearch_auth_config_file_path = "/mnt/data/configs/es-auth.cfg"
+pds_nucleus_opensearch_url                   = "https://search-node2-dev-abcdefghijklmnop.us-west-2.es.amazonaws.com:443"
+pds_node_name                                = "PDS_SBN"
+pds_nucleus_harvest_replace_prefix_with      = "s3://pds-nucleus-staging"
+
+mwaa_dag_s3_bucket_name         = "pds-nucleus-airflow-dags-bucket-mcp-test"
+pds_nucleus_staging_bucket_name = "pds-nucleus-staging-mcp-test"
```

Comment:
how is this bucket different from pds-test-nucleus-staging ?

Reply:
There is no difference. This bucket is created by terraform, so we can execute the scripts in a clean environment and deploy Nucleus from scratch, without asking admins to manually create buckets.